### DISCUSSION TITLE
error catch

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#5608] [Questions] [answered]
I'm confused...  Why i can't catch this error? 

![error](https://user-images.githubusercontent.com/56837461/155192024-de7f4b75-7478-4bdd-a45c-b5013dced61a.png)

Error is on purpose, just have to spread values for no error.

**Comment by GaryAustin1** [comment, marked as answer]
You would need to throw an error to get "handled" errors to the catch.
Here is a decent discussion of what Supabase does with errors:
https://github.com/supabase/supabase-js/issues/92

**Reply by EdgarsLv** [reply]
ty
